flex-templates/python/regional_dlp_transform/bigquery_dlp_bigquery.py

```diff
@@ -265,8 +277,9 @@ def expand(self, pcoll):
                 GoogleCloudOptions).project
         if self.project is None:
             raise ValueError(
-                'GCP project name needs to be specified '
-                'in "project" pipeline option')
+                'GCP project name needs to be specified'
```

Comment:
I think you need the space after specified else the help might look like `specifiedin "dlp_project..."` but plz verify

Reply:
I think the space is really needed. Thanks!